build.gradle

```diff
@@ -66,11 +67,12 @@ buildscript {
         classpath 'com.android.tools.build:gradle:7.0.3'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${versions.kotlin}"
         classpath "org.jlleitschuh.gradle:ktlint-gradle:10.0.0"
+        classpath "org.jacoco:org.jacoco.core:${versions.jacoco}"
     }
 }
 
 project.ext {
-    testCoverage = project.hasProperty('testCoverage')
```

Comment:
with what was before you could actually just pass an argument to gradlew wrapper (in this case `-PtestCoverage=true`) so i would go against this change (for easier configurability)

Reply:
Rolled back to previous solution. Just to confirm, now running `./gradlew` with `-PtestCoverage=false` also generates reports, because previous logic checks only for parameter presence, but I think we can leave it like that, because, as you said, it is only for easier configurability.